frontend/src/v5/store/tickets/tickets.types.ts

```diff
@@ -212,8 +213,11 @@ export type OverridesDicts = {
 	transparencies: MeshIdTransparencyDict,
 };
 
-export type ITicketsFilters = {
-	complete: boolean,
-	templates: string[],
-	queries: string[],
+export type FilterTypesByProperty = {
+	[property in string]: {
+		[type in CardFilterType]?: BaseFilter;
+	};
+};
+export type FilterPropertiesByModule = {
+	[module in string]?: FilterTypesByProperty;
```

Comment:
I think the typing of this is a bit complex. Instead of this you could have an object that is more like
```
{
  'module.mycoolProp.number' : {operator: 'eq', values: [1]},
  'module.mycoolProp.string' : {operator: 'eq', values: ['brown', 'yellow']}
}
```
Maybe you could use templates literals to enforce a format?
https://www.typescriptlang.org/docs/handbook/2/template-literal-types.html